src/databases/functions/users/get-user-by-user-id.js

```diff
@@ -1,12 +1,14 @@
-const { User } = require("../../models")
+const {User} = require('../../models');
 
 module.exports = async (userId) => {
-    if(userId == null) return;
-    let user = await User.findOne({
-        where: {
-            user_id: userId
-        }
-    })
-    if(user == null) return []
-    return user
-}
+  if (userId == null) return;
+  const user = await User.findOne({
+    where: {
+      user_id: userId
+    },
+    raw: true
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Returning raw data from the database query can be a good performance optimization. However, it also means that you lose all the instance methods and other benefits provided by Sequelize models. Make sure this change doesn't break any code that expects a Sequelize model instance.


<!-- This is an auto-generated comment by OSS CodeRabbit -->